atd-vzd/migrations/migration_update_audit_log_trigger_2021_07_15--1402.sql

```diff
@@ -0,0 +1,170 @@
+       + (NEW.atd_fatality_count * (estCompCostList[2]))
+       + (NEW.sus_serious_injry_cnt * (estCompCostList[3]))
+       + (NEW.nonincap_injry_cnt * (estCompCostList[4]))
+       + (NEW.poss_injry_cnt * (estCompCostList[5]))
+       + (NEW.non_injry_cnt * (estCompCostList[6]))
+    )::decimal(10,2);
+
+    IF (NEW.atd_fatality_count > 0) THEN NEW.est_comp_cost_crash_based = estCompCostListCrashBased[1];
+    ELSIF (NEW.sus_serious_injry_cnt > 0) THEN NEW.est_comp_cost_crash_based = estCompCostListCrashBased[2];
+    ELSIF (NEW.nonincap_injry_cnt > 0) THEN NEW.est_comp_cost_crash_based = estCompCostListCrashBased[3];
+    ELSIF (NEW.poss_injry_cnt > 0) THEN NEW.est_comp_cost_crash_based = estCompCostListCrashBased[4];
+    ELSIF (NEW.non_injry_cnt > 0) THEN NEW.est_comp_cost_crash_based = estCompCostListCrashBased[5];
+    ELSIF (NEW.unkn_injry_cnt > 0) THEN NEW.est_comp_cost_crash_based = estCompCostListCrashBased[6];
+    ELSE NEW.est_comp_cost_crash_based = 0;
+    END IF;
+
+    NEW.est_econ_cost = (0
+       + (NEW.unkn_injry_cnt * (estCompEconList[1]))
+       + (NEW.atd_fatality_count * (estCompEconList[2]))
+       + (NEW.sus_serious_injry_cnt * (estCompEconList[3]))
+       + (NEW.nonincap_injry_cnt * (estCompEconList[4]))
+       + (NEW.poss_injry_cnt * (estCompEconList[5]))
+       + (NEW.non_injry_cnt * (estCompEconList[6]))
+    )::decimal(10,2);
+    --- END OF COST OPERATIONS ---
+
+    ------------------------------------------------------------------------------------------
+    -- SPEED MGMT POINTS
+    ------------------------------------------------------------------------------------------
+    speedMgmtList = ARRAY (SELECT speed_mgmt_points FROM atd_txdot__speed_mgmt_lkp ORDER BY speed_mgmt_id ASC);
+
+    NEW.speed_mgmt_points = (0
+       + (NEW.unkn_injry_cnt * (speedMgmtList [1]))
+       + (NEW.atd_fatality_count * (speedMgmtList [2]))
+       + (NEW.sus_serious_injry_cnt * (speedMgmtList [3]))
+       + (NEW.nonincap_injry_cnt * (speedMgmtList [4]))
+       + (NEW.poss_injry_cnt * (speedMgmtList [5]))
+       + (NEW.non_injry_cnt * (speedMgmtList [6]))
+    )::decimal (10,2);
+    --- END OF SPEED MGMT POINTS ---
+
+    ------------------------------------------------------------------------------------------
+    -- MODE CATEGORY DATA
+    ------------------------------------------------------------------------------------------
+    NEW.atd_mode_category_metadata = get_crash_modes(NEW.crash_id);
+    --- END OF MODE CATEGORY DATA ---
+
+    -- Record the current timestamp
+    NEW.last_update = current_timestamp;
+    RETURN NEW;
+END;
+$$;
+
+-- set ownership if needed
+alter function atd_txdot_crashes_updates_audit_log() owner to atd_vz_data;
+
+-- replace the trigger to invoke the function on upsert
+create trigger atd_txdot_crashes_audit_log before insert or update on
+    public.atd_txdot_crashes for each row execute function atd_txdot_crashes_updates_audit_log();
```

Comment:
We need to make sure we update this file:
https://github.com/cityofaustin/atd-vz-data/blob/master/atd-vzd/triggers/atd_txdot_crashes_updates_audit_log.sql

And incorporate Mateo's changes:
https://github.com/cityofaustin/atd-vz-data/pull/1007/files